src/modules/tableInput.js

```diff
@@ -0,0 +1,22 @@
+document.getElementById('submitButton').addEventListener('click', () => {
+  const input1 = document.querySelector('#input1');
+  const input2 = document.querySelector('#input2');
+
+  const data = {
+    user: input1.value,
+    score: input2.value,
+  };
+
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/1000/scores/', {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  })
```

Comment:
As requested on the project requirements:
![image](https://user-images.githubusercontent.com/42799579/196830159-6dd5c225-75f3-46eb-9cab-bfa4d5c4117f.png)
You need to use _async_ and _await_ to consume your API. This means that your fetch should happen _asyncronously_ to your code, not at the same time as the normal logic. To fix this, please use an _async_ function and _await_ for your fetch response 😄.

You can do something like this:
const myFunction = async () => {
const response = await fetch( ... )
_do stuff with the response_
}

If you are interested, I think this [article](https://dmitripavlutin.com/javascript-fetch-async-await/) can help you 😉.